openmp/runtime/src/kmp.h

```diff
@@ -1847,6 +1848,14 @@ typedef struct kmp_sched_flags {
   unsigned contains_last : 1;
 #if KMP_USE_HIER_SCHED
   unsigned use_hier : 1;
+#if KMP_AFFINITY_SUPPORTED && KMP_STATIC_STEAL_ENABLED
+  unsigned use_hybrid : 1;
+  unsigned unused : 27;
+#else
+  unsigned unused : 28;
+#endif
+#elif KMP_AFFINITY_SUPPORTED && KMP_STATIC_STEAL_ENABLED
+  unsigned use_hybrid : 1;
```

Comment:
Is it really worth adding all these cases for 1 bit?

Reply:
Removed the macros here and put comments for relevant macros